dapps/src/lib.rs

```diff
@@ -205,6 +207,11 @@ impl Server {
 	pub fn set_panic_handler<F>(&self, handler: F) where F : Fn() -> () + Send + 'static {
 		*self.panic_handler.lock().unwrap() = Some(Box::new(handler));
 	}
+
+	/// Returns address that this server is bound to.
+	pub fn addr(&self) -> &SocketAddr {
+		self.server.as_ref().unwrap().addr()
```

Comment:
what's the proof that `server` is `Some`? I also notice that this is only used for tests, maybe it should be marked `#[cfg(test)]`?